index.js

```diff
@@ -1,14 +0,0 @@
-import { default as chromedriver } from './lib/chromedriver';
```

Comment:
I wanted to confirm we are not going to have the same issue as in https://github.com/appium/node-teen_process/pull/253 if we do not expose index.js

Reply:
unrelated; that problem was because there was a `noEmit` in `tsconfig.json`